### ISSUE TITLE
YOLOv3

### ISSUE THREAD

**Issue by iwasakishuto** [iwasakishuto/Papers-With-Comments#5]
# YOLOv3
- An Incremental Improvement
- Submitted on Sun, 8 Apr 2018 (v1)

### Link
- [paper](https://arxiv.org/pdf/1804.02767.pdf)
- [Project Webpage](https://pjreddie.com/darknet/yolo/)
- [TED Talk](https://www.ted.com/talks/joseph_redmon_how_computers_learn_to_recognize_objects_instantly)

### Summary
- They **made a bunch of little design changes** to make it better.
- Little **bigger than last time** but more accurate.
- It is **still fast**.

### Specific Content
- **Bounding Box Prediction**:
    - Their system predicts bounding boxes using **dimension clusters as anchor boxes**.
    - Predicts an objectness score for each bounding box using **logistic regression**.
    - Assign one bounding box prior for each ground truth object.
- **Class Prediction**:
    - Each box predicts the classes (the bounding box may contain) using multilabel classification.
    - Do not use a softmax and use logistic classifications.
    - During training they use binary cross-entropy loss for the class predictions.
- **Predictions Across Scales**:
    - YOLOv3 predicts boxes at 3 different scales and extracts features from those scales using a similar conceot to feature pyramid network.
- **Feature Extractor**:
    - Use a new hybrid (YOLOv2 + Darknet-19 + additional layers → Darknet-53) network for performing feature extraction.
    - Achieve the highest measured floating point operations per second.

### Things They Tried That Didn't Work
- **Anchor box x,y offset predictions** → Decreased model stability and didn't work very well.
- **Linear x,y predictions instead of logistic** → Couple point drop in mAP.
- **Focal loss** → Drop mAP about 2 points
- **Dual IOU thresholds and truth assignment** → Could not get good results.
